distribution/docs/src/main/resources/content/_coreConcepts/data-standards.adoc

```diff
@@ -0,0 +1,77 @@
+:type: subCoreConcept
+:status: published
+:title: Data Formats
+:parent: Standards Supported by ${branding}
```

Comment:
I think we should define what "support" means.

Does it mean?
- these are the formats that can be converted into metacards?
- these are the format that can be ingested by every endpoint that supports ingest?
- these are the formats that can be ingested from the command line?
- all metacards can be converted to these (and only these)?
- all metacards can be converted to these and other formats, but these are the most common?
- all endpoint can provide data in these formats?

Deconstructing the questions, I see three dimension:
- Endpoint (SOAP 1.3, SOAP 2.0, JSON, CSW, OpenSearch, DDF Console...)
- Activity (ingest v. query)
- Format (JPEG, PDF, XML, Text, OOXML, GeoJSON, etc.)

When talking about "supported", we might have to specify Endpoint & Activity.

Lastly, there is room for shades of gray (is OOXML just a kind of XML? No, it really is something special because it is the format used by PowerPoint and Word). We'll might have to kick these things around to come to consensus. 

Maybe a meeting around a table and projector could move us along faster?

Reply:
I'd say whiteboard even more than projector